write_column.go

```diff
@@ -14,20 +15,13 @@ type ColumnarStatement interface {
 }
 
 func (stmt *stmt) ColumnWriter() ColumnWriter {
-	stmt.ch.data.reserveColumns()
-	return stmt.ch.data
+	stmt.ch.block.Reserve()
+	return stmt.ch.block
 }
 
 func (stmt *stmt) ColumnWriterEnd(rows uint64) error {
-	writtenBlocks := stmt.counter / stmt.ch.blockSize
-	stmt.counter += int(rows)
-	stmt.ch.data.numRows += rows
-
-	var err error
-	if stmt.counter/stmt.ch.blockSize > writtenBlocks {
-		err = stmt.ch.data.write(stmt.ch.serverRevision, stmt.ch.conn)
-	}
-	return err
```

Comment:
Actually, it might be better to just allow multiple blocks so caller can control both size and write them concurrently with the columnar interface, like:

```
b1 := stmt.Block() // Default "ch.block"
b2 := stmt.CreateBlock() // Create extra block and return it
wg := sync.WaitGroup()
wg.Add(2)
go func() {
  b1.WriteUInt8(0, 42)
  wg.Done()
}
go func() {
  b2.WriteUInt8(0, 43)
  wg.Done()
}
wg.Wait()
stmt.Commit() // Write both blocks, followed by an empty block to signalise EOF
```

What do you think? @kshvakov